lib/errors.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const errorTitle = (code) => `${code}_ERROR`;
+const errorMsg = (code) => `There is a ${code}_ERROR`;
+
+const getContent = (err, debug, translate) => {
+
+  const content = {};
```

Comment:
Doesn't really matter for this PR. But there is probably question here about our house style and whether we want to bother making objects const. If it just makes the reference const but not the object itself I wonder if it could be misleading. Does anyone know if it is possible to have a truly immutable object in ES6?

It feels weird to me to have const all over the place and then have a load of mutation going on in objects because ES6 doesn't stop it.

Could we set "content" once and then leave it alone? Is that feasible? Does anyone but me care? ;-)